public/locales/it/home.json

```diff
@@ -9,11 +9,11 @@
         },
         "indications": {
             "landscape": "Dopo aver inviato il formulario, ti consigliamo di impostare il tuo dispositivo in modalità paesaggio. Potrai vedere meglio il tuo familiare.",
```

Comment:
if we don't use `landscape` translation anymore we can remove it

Reply:
ok, done